inc/entity.class.php

```diff
@@ -512,6 +547,64 @@ public static function cronCreditExpired($task)
         return 1;
     }
 
+    public static function cronLowCredits($task)
+    {
+        global $CFG_GLPI, $DB;
+
+        if (!$CFG_GLPI['use_notifications']) {
+            return 0;
+        }
+
+        $alert = new Alert();
+        $credits_iterator = $DB->request(
+            [
+                'SELECT' => [
+                    'glpi_plugin_credit_entities.*',
+                    new QueryExpression('SUM(glpi_plugin_credit_tickets.consumed) AS quantity_consumed')
```

Comment:
SUM does not seem to be exploited, only id and name are.
```suggestion
                    'glpi_plugin_credit_entities.id',
                    'glpi_plugin_credit_entities.name',
```